Python/Product/PythonTools/PythonTools/Environments/AddExistingEnvironmentView.cs

```diff
@@ -508,24 +508,32 @@ ProjectView selectedProject
                         version = null;
                     }
 
-                    factory = SelectedProject.Node.AddMSBuildEnvironment(
-                        RegistryService,
-                        PrefixPath,
-                        InterpreterPath,
-                        WindowsInterpreterPath,
-                        PathEnvironmentVariable,
-                        version,
-                        InterpreterArchitecture.TryParse(ArchitectureName ?? ""),
-                        Description
-                    );
+                    if (SelectedProject.Node != null) {
```

Comment:
Some other places also check SelectedProject for null.

Reply:
We're inside a `if (SelectedProject != null)` here.